docs/application/native/guides/applications/ui-app.md

```diff
@@ -5,20 +5,14 @@ UI applications are Tizen native applications with a graphical user interface th
 
 A Tizen native UI application is similar to a conventional Linux application, with some additional features optimized for mobile and wearable devices. The additional features have constraints, such as a relatively small screen size and lack of system resources compared to a larger system. For example, for power management reasons, the application can take actions to reduce usage when it finds out that it has its display window covered over by another application window. State change events are delivered to make it possible to detect these situations.
 
-When creating a native UI application, you can select between EFL and DALi UI frameworks:
+When creating a native UI application, you can see EFL UI frameworks:
```

Comment:
Is it see or select?
How about this?

 When creating a native UI application, you can use EFL UI frameworks:

Reply:
Actually, it is 'see', but I accepted your proposal. :)